documentation/man/features/build.rst

```diff
@@ -70,6 +71,10 @@ OPTIONS
   compilation/linking tasks. You can enable it by default via `use_llvm` option
   in the `build.config` file.
 
+-c, \--clean:
+  The clean option removes most files generated by the kernel build system but
+  keeps the kernel configuration.
```

Comment:
Maybe this:

The clean option removes files generated by the kernel build system but keeps the kernel configuration. This option considers whether the user is using or not kw env.